edge/browsersupport-chrome.js

```diff
@@ -0,0 +1,243 @@
+/* global chrome: false, msBrowser: false */
+
+/* Edge Support - PREVIEW */
+
+if (typeof msBrowser !== 'undefined') {
```

Comment:
Same thing here -- make sure RESenvironment is loaded before browsersupport-chrome.js, then refactor this shim into `var chrome = RESenvironment.navigatorApiObject();` .. this idea needs a little polish.